packages/suite/src/support/messages.ts

```diff
@@ -5000,6 +5005,14 @@ export default defineMessages({
         defaultMessage:
             'Access the NFTs stored in your wallet. Currently available for EVM-based chains only.',
     },
+    TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT: {
+        id: 'TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT',
+        defaultMessage: 'Support for Ethereum L2 networks (Optimism, Arbitrum One, Base)',
+    },
+    TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT_DESCRIPTION: {
+        id: 'TR_EXPERIMENTAL_ETHEREUM_L2_SUPPORT_DESCRIPTION',
+        defaultMessage: 'Access your Optimism, Arbitrum One, and Base accounts.',
```

Comment:
I think we do not use `my`, `your` in the app

So please change it to something like `Allows to enable Base, Optimism and Arbitrum one networks.`

Reply:
I'll say that it enables discovery and creation of those accounts

Reply:
just discovery is enough I think, the accounts "exist", we do not create them, I will change it and merge it. thanks

Reply:
fixed